base_force_record_noupdate/models/res_config_settings.py

```diff
@@ -0,0 +1,128 @@
+        help="Set the default models that should have noupdate=True.",
+    )
+    models_force_noupdate = fields.Many2many(
+        "ir.model",
+        string="Models to force noupdate=True",
+        domain=[("transient", "=", False)],
+    )
+
+    def set_values(self):
+        """
+        Save the selected models in the system parameters as a comma-separated list.
+        """
+        res = super(ResConfigSettings, self).set_values()
+        # get current values from the system parameters and set noupdate=False
+        current_models_force_noupdate = (
+            self.env["ir.config_parameter"].sudo().get_param("models_force_noupdate")
+        )
+        if current_models_force_noupdate:
+            self._compare_values(current_models_force_noupdate)
+        # save the selected models in the system parameters as a comma-separated list
+        if self.models_force_noupdate:
+            new_models_force_noupdate = ",".join(
+                map(str, self.models_force_noupdate.ids)
+            )
+            self.env["ir.config_parameter"].sudo().set_param(
+                "models_force_noupdate", new_models_force_noupdate
+            )
+            self._update_ir_model_data(self.models_force_noupdate)
+        else:
+            self.env["ir.config_parameter"].sudo().set_param(
+                "models_force_noupdate", False
+            )
+        return res
+
+    def _compare_values(self, current_models_force_noupdate):
+        # convert the current values from the system parameters to a list of model IDs
+        current_models_force_noupdate = list(
+            map(int, current_models_force_noupdate.split(","))
+        )
+        if self.models_force_noupdate:
+            self._update_ir_model_data(
+                self.env["ir.model"].browse(
+                    list(
+                        set(current_models_force_noupdate)
+                        - set(self.models_force_noupdate.ids)
+                    )
+                ),
+                noupdate=False,
+            )
+        else:
+            self._update_ir_model_data(
+                self.env["ir.model"].browse(current_models_force_noupdate),
+                noupdate=False,
+            )
+
+    def get_values(self):
+        """
+        Get the selected models from the system parameters as a comma-separated list.
+        """
+        res = super(ResConfigSettings, self).get_values()
+        # get the selected models from the system parameters as a comma-separated list
+        models_force_noupdate = (
+            self.env["ir.config_parameter"].sudo().get_param("models_force_noupdate")
+        )
+        # convert the comma-separated list in a list of model IDs
+        if models_force_noupdate:
+
+            models_force_noupdate = self.env["ir.model"].browse(
+                list(map(int, models_force_noupdate.split(",")))
+            )
+            res.update(models_force_noupdate=models_force_noupdate.ids)
```

Comment:
This does not seem to work: on my test database I have 	95,206,81,8,36 in the system parameter, yet when I open the settings, I only see record rules

![image](https://github.com/OCA/server-tools/assets/2032807/ea4040f4-f3a8-4fec-82e9-89d93a79773f)

I think you need to send a Command.link: ``
```suggestion
            res.update(models_force_noupdate=[Command.link(id) for id in models_force_noupdate.ids])
```
(warning: untested)